laravel-app/app/Services/PostService.php

```diff
@@ -25,10 +24,7 @@ public function createPost(User $user, array $validatedData): UserPost
 
     /**
      * To update an existing post.
-     *
-     * @param UserPost $post
      * @param array<string, mixed> $validatedData
-     * @return bool
      */
```

Comment:
removee!!!! @

Reply:
Apologies sir, but if I remove the @param for the arrray the larastan checker throws error and requires that to be added. 🙇